packages/hooks/src/index.ts

```diff
@@ -1,9 +1,9 @@
 export { default as useAccountTransferVisible } from './useAccountTransferVisible';
-export { default as useCfdAccounts } from './useCfdAccounts';
+export { default as useCashierLocked } from './useCashierLocked';
+export { default as useCFDAccounts } from './useCFDAccounts';
```

Comment:
@hamid-deriv `packages/hooks/src/useCfdAccounts.ts` is wrong, Better to reset with mster

Reply:
There were two files with the same name but a different type of case!